clients/unshaded/src/main/java/tachyon/client/block/RemoteBlockInStream.java

```diff
@@ -37,11 +47,25 @@
    * @param blockId the block id
    * @param blockSize the block size
    * @param location the location
+   * @throws IOException if I/O error occurs
```

Comment:
Could you be more specific here? (For example, we throw an IOException when the block is not on the worker)